wormchain/app/test_helpers.go

```diff
@@ -0,0 +1,260 @@
+package app
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
```

Comment:
lets move this file under the `apptesting` folder as well?

Reply:
Yes, moving to apptesting folder was possible and is a better fit than the app package. Thanks!